catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transaction/DeleteActionImpl.java

```diff
@@ -17,16 +17,22 @@
 import net.opengis.cat.csw.v_2_0_2.DeleteType;
 import net.opengis.cat.csw.v_2_0_2.QueryConstraintType;
 import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.spatial.ogc.csw.catalog.actions.DeleteAction;
 import org.codice.ddf.spatial.ogc.csw.catalog.common.CswConstants;
 
-/** A DeleteAction represents a single delete action within a CSW transaction. */
-public class DeleteAction extends CswAction {
+/** A DeleteActionImpl represents a single delete action within a CSW transaction. */
+public class DeleteActionImpl implements DeleteAction {
+
   private QueryConstraintType queryConstraintType;
 
   private Map<String, String> prefixToUriMappings;
```

Comment:
The original plan was to remove this since `getPrefixToUriMappings()` wasn't being used as far as I could tell. I decided to leave it in the interface anyway though since it's just a refactoring (the functionality shouldn't change) and the benefit of removing it (cleaning up a small bit of code) wasn't worth the risk of possibly breaking downstream projects (aside from the fact that they would have to convert to using the new interface and impl class).